src/main/webapp/go.js

```diff
@@ -105,7 +108,11 @@ function addScriptToHead() {
  * to reflect the current state of the hunt.
  */
 function getHunt() {
-  fetch(DATA_URL).then((response) => response.json()).then((mssg) => {
+  // ID of the scavenger hunt that the user is on,
+  // in the form of: hunt_id=ScavengerHunt([ID number here])
```

Comment:
Does this mean the URL literally looks like hunt_id=ScavengerHunt(123)?  Why do you need the ScavengerHunt part?  Can this just be hunt_id=123?

Reply:
This is mostly for simplicity; I believe that entity.getKey() returns the string "ScavengerHunt([some number here])."

Reply:
But then won't GoDataServlet try to parse ScavengerHunt(123) as a number?  Won't that not return what you expect?

Reply:
GoDataServlet currently tries to parse ScavengerHunt([number]) as a string, which is used to find the corresponding entity in Datastore.

Reply:
Oh I see, my mistake.  I think it would be better to just pass around the ID, which you could then use as a filter when you are querying datastore.  But for the sake of simplicity, I think this is fine.

Reply:
That makes sense -- I actually just changed huntID to a long instead of a String in the latest commit; turns out that it's easier to efficiently retrieve an entity from Datastore by key using the ID number.